normits_demand/models/notem/attraction_models.py

```diff
@@ -79,45 +80,37 @@ class HBAttractionModel(HBAttractionModelPaths):
 
     # Define wanted columns
     _target_col_dtypes = {
-        'employment': {
-            'msoa_zone_id': str,
-            'sic_code': int,
-            'people': float
-        },
-        'trip_weight': {
-            'purpose': int,
-            'mode': int,
-            'sic_code': int,
-            'trip_rate': float
-        },
+        "employment": {"msoa_zone_id": str, "sic_code": int, "people": float},
+        "trip_weight": {"purpose": int, "mode": int, "sic_code": int, "trip_rate": float},
     }
 
     # Define segmentations used
     _segmentations = {
-        'employment': 'sic',
-        'trip_weights': 'hb_p_m_sic',
-        'pure_attractions': 'hb_p_m',
+        "employment": "sic",
+        "trip_weights": "hb_p_m_sic",
+        "pure_attractions": "hb_p_m",
     }
 
     # Define segment renames needed
     _seg_rename = {
-        'sic_code': 'sic',
-        'purpose': 'p',
-        'mode': 'm',
+        "sic_code": "sic",
+        "purpose": "p",
+        "mode": "m",
     }
 
-    _zoning = 'msoa'
-
-    def __init__(self,
-                 employment_paths: Dict[int, nd.PathLike],
-                 production_balance_paths: Dict[int, nd.PathLike],
-                 trip_weights_path: str,
-                 non_resi_path: str, # TODO Remove temp parameter replacement for land use
-                 export_home: str,
-                 balance_zoning: nd.BalancingZones = None,
-                 constraint_paths: Dict[int, nd.PathLike] = None,
-                 process_count: int = consts.PROCESS_COUNT
-                 ) -> None:
+    _zoning = "msoa"
+
+    def __init__(
+        self,
+        employment_paths: Dict[int, nd.PathLike],
+        production_balance_paths: Dict[int, nd.PathLike],
+        trip_weights_path: str,
+        non_resi_path: str,  # TODO Remove temp parameter replacement for land use
```

Comment:
It's from when we were using non resi data instead of land use employment and it would be removed once the land use model is updated to produce data in the new non resi format, at that point it could just be removed.

Reply:
To clarify, this isn't a Dlog specific thing? This is more about getting the non-resi land use into NoTEM attractions?

Is this to prevent what was previously a hard-coded path to avoid this trouble?

Reply:
It wasn't added for D-Log but the parameter to choose between it in `NoTEM.run` was added because DLog uses the other way of providing the input path. DLog puts the employment land use files in the same folder structure / filename as the land use model but saves the files in the new format so once the Land use model is updated and NoTEM reverts back to the old way of providing the path DLog will be doing the same thing as NoTEM.

Reply:
New issue #981